crawler/models/ipad_model.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+from pydantic import BaseModel, field_validator
+
+
+class IpadItemModel(BaseModel):
+    name: str
+    params: str
+    price: str
+    downloaded_date: str
+    availability: str
+    url: str
+    image_url: str
+
+    @field_validator('price')
+    def clean_price(cls, v):
+        return v.replace('\xa0', '').replace('€', '').replace(' ', '').strip()
+
+    @field_validator('availability')
```

Comment:
sitos valymo funkcijos dubliuojasi per modelius,cia yra blogai,geriau gal turt funkciju  sarasa kazkur  kur is ten jas kviesi,nes dabar tapati funkcija kuri kiekviename modelyje